wcfsetup/install/files/lib/system/language/preload/command/CachePreloadPhrases.class.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace wcf\system\language\preload\command;
+
+use Laminas\Diactoros\Stream;
+use wcf\data\language\Language;
+use wcf\system\event\EventHandler;
+use wcf\system\language\preload\PreloadPhrase;
+use wcf\system\language\preload\PreloadPhrasesCollecting;
+use wcf\util\StringUtil;
+
+/**
+ * Rebuilds the phrase preload cache for the
+ * requested language.
+ *
+ * @author Alexander Ebert
+ * @copyright 2001-2022 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package WoltLabSuite\Core\System\Language\Preload\Command
+ * @since 6.0
+ */
+final class CachePreloadPhrases
+{
+    private readonly EventHandler $eventHandler;
+    private readonly Language $language;
+
+    public function __construct(Language $language)
+    {
+        $this->eventHandler = EventHandler::getInstance();
+        $this->language = $language;
+    }
```

Comment:
```suggestion
    public function __construct(Language $language)
    {
        $this->language = $language;

        $this->eventHandler = EventHandler::getInstance();
    }
```

Visually split the actual parameters from the dependencies. Move the dependencies to the bottom, because this is a more natural order.